fse/bitreader.go

```diff
@@ -64,7 +69,8 @@ func (b *bitReader) fillFast() {
 		return
 	}
 	// Do single re-slice to avoid bounds checks.
```

Comment:
Is this comment still correct?

Reply:
I guess it never was ;)

The compiler cannot see that if `b.in[b.off-4 : b.off]` passes the bounds check then `len(v) == 4`.

At least now only 2 checks are needed. I guess something changed in the compiler. Pretty sure I checked the assembly for this a long time ago. At least now it is 2 bounds checks and a `MOVL`.

Reply:
cool. thanks for checking